backend/qemu.pm

```diff
@@ -681,7 +681,9 @@ sub start_qemu ($self) {
     elsif ($vars->{OFW}) {
         $use_usb_kbd = $self->qemu_params_ofw;
     }
-    sp('vga', $vars->{QEMUVGA}) if $vars->{QEMUVGA};
+    # qxl hard-codes the default resolution, see poo#111992
+    die "qxl is unsupported\n" if ($vars->{QEMUVGA} // '') eq 'qxl';
```

Comment:
A fatal `die` would overshoot the goal. "Unsupported" doesn't mean "not allowed at all". As shown in https://github.com/os-autoinst/os-autoinst/pull/2082#issuecomment-1152199464 it is also not *that* bad, e.g. if one would get past the bootloader it would actually work.

Reply:
Well, we don't know what's installed and wether the test has needles for the right resolution.